packages/pinned_pool/src/pinned_pool.rs

```diff
@@ -822,6 +822,12 @@ impl<'s, T> PinnedPoolInserter<'s, T> {
     ///
     /// The closure must initialize the `MaybeUninit<T>` before returning.
     ///
+    /// # Panics
```

Comment:
Panic-safety can be assumed as provided by every API that accepts callbacks - there is no need to document it.